### ISSUE TITLE
Feature request about View Asset page and Check out rules

### ISSUE THREAD

**Issue by izombielandgit** [grokability/snipe-it#9863]
**Describe the solution you'd like**
There are two feature requests.

1. View Asset page (as https://xxx.snipeit/hardware/123). If I scan the QR code from my mobile phone to open the url, it will ask to log in. 
Is it possible to display some basic information of the asset before logging in?(like Name, Tag, Checked Out To, Location or the admin user can customize which field can be displayed)

2. When an asset checked out to one person needs to be checked out to another person. Current system can only check in first and then check out.
Is it possible to check out directly from one person to another.

I don't know if that's clear, or whether you would consider these requests.  thank you for your great work anyway.

**Comment by snipe** [comment]
I don’t think we could do that without opening up a ton of security implications :(
